<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>КупиДом</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Registration_Widget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Регистрация</string>
     </attribute>
     <widget class="QLabel" name="label_registration">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
     <widget class="QLabel" name="User_name_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>имя пользователя</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username_input_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Password_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>придумайте пароль</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_input_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Age_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите ваш возраст</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="age_input_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLabel" name="Password_confirm_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>подтвердите пароль</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_confirm_input_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Income_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Укажите ваш доход в месяц</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="income_input_registration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Registration_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Зарегестрироваться</string>
      </property>
     </widget>
     <widget class="QLabel" name="money_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите вашу сумму денег</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Current_money_input_registrarion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Вход</string>
     </attribute>
     <widget class="QLabel" name="label_login">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username_input_login_checker">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_input_login_checker">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="User_name_login">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя пользователя</string>
      </property>
     </widget>
     <widget class="QLabel" name="Password_login">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Login_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
